pkg/pool-utils/test/ProtocolFeeCache.test.ts

```diff
@@ -75,7 +75,11 @@ describe('ProtocolFeeCache', () => {
     it('reverts during deployment', async () => {
       await expect(
         deploy('MockProtocolFeeCache', {
-          args: [vault.protocolFeesProvider.address, { swap: 137, yield: ProtocolFee.SWAP, aum: ProtocolFee.YIELD }],
+          args: [
+            vault.address,
+            vault.protocolFeesProvider.address,
+            { swap: 137, yield: ProtocolFee.SWAP, aum: ProtocolFee.YIELD },
+          ],
```

Comment:
I would add one small getter test here and in `RecoveryMode.test.ts` besides the ones you've added in base pools.

Reply:
There isn't really an independent RecoveryMode test, and I don't think the contortions to make one are worth it. But I can add it to ProtocolFeeCache at least.